### PR TITLE
Addressed most CSBS journal referees feedback on CWP analysis chapter

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib [hunk 1]
```diff
@@ -2372,3 +2371,13 @@ @article{1742-6596-664-8-082046
   url={http://stacks.iop.org/1742-6596/664/i=8/a=082046},
   year={2015},
 }
+
+@online{Spark,
```
Comment (davidlange6) [inline CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib]: I guess the web site is incorrect below (I think the paper I added should also be referenced)

Comment (eduardo-rodrigues) [reply CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib]: From looking at your PR, yes, your paper is a good citation to have. But why do you say https://www.scala-lang.org/ is wrong?

**Comment by davidlange6** [comment]
> On Nov 8, 2018, at 12:47 PM, Eduardo Rodrigues <notifications@github.com> wrote:
> 
> @eduardo-rodrigues commented on this pull request.
> 
> In CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib:
> 
> > @@ -2372,3 +2371,13 @@ @article{1742-6596-664-8-082046
>    url={http://stacks.iop.org/1742-6596/664/i=8/a=082046},
>    year={2015},
>  }
> +
> +@online{Spark,
> 
> From looking at your PR, yes, your paper is a good citation to have. But why do you say https://www.scala-lang.org/ is wrong?
> 

because spark and scala are two different things.
eg, https://spark.apache.org if you want some generic spark website reference.

File: CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib (hunk 1 above)
Comment (eduardo-rodrigues) [reply CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib]: Oh, I managed to do a wrong copy-and-paste somehow! Thanks for spotting this.

Comment (eduardo-rodrigues) [reply CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib]: I will get this fixed after lunch.

**Comment by eduardo-rodrigues** [comment]
From my side this PR is ready. @davidlange6, do you want to finalise your review, to try and also get this merged asap? Thanks.

**Comment by jouvin** [comment]
I have the feeling this PR is ready to be merged, except if I missed something. In this case, I suggest to do it asap so that we can solve potential conflicts with the 2 remaining ones with some pending issues.

**Comment by eduardo-rodrigues** [comment]
OK, 2 happy people (+me ;-)) … I'm merging. Thanks for the ckecks.

**Comment by graeme-a-stewart** [comment]
Count me as happy too 👍

**Comment by eduardo-rodrigues** [comment]
Thanks :-).

**Pull request merged**